pkgs/shells/zsh/setxkbmap-completion-xdg-data-dirs.patch

```diff
@@ -0,0 +1,18 @@
+diff --git a/Completion/X/Command/_setxkbmap b/Completion/X/Command/_setxkbmap
+index f7310ec..8594e2a 100644
+--- a/Completion/X/Command/_setxkbmap
++++ b/Completion/X/Command/_setxkbmap
+@@ -10,9 +10,10 @@ _setxkbmap() {
+ 
+     # xkb files may be in different places depending on system
+     local dir sourcedir
+-    for dir in /usr/lib/X11/xkb /usr/share/X11/xkb; do
+-        if [ -d $dir ] ; then
+-           sourcedir=$dir
++    for dir in $(echo ${XDG_DATA_DIRS:gs/:/ /}); do
```

Comment:
This is bash gibberish to me - I'm assuming you got this from someone else who traverses `XDG_DATA_DIRS` sensibly in bash?

Reply:
you're right. The thing is, in bash `IFS=":"` would solve the problem. However this is `zsh` and I didn't manage to get this approach working, sorry :/
Do you have a better/cleaner idea? :)

Reply:
I have no idea I'm afraid - surely there's someone else we can copy? I'm just aware that this sort of thing is easy to do wrong.

Reply:
I just learned that `IFS` can be used in zsh if `setopt sh_word_split` is set. Do you prefer the current change over the original solution?
If so, file it upstream, but tomorrow, I need to catch some sleep now.